contracts/SSVNetwork.sol

```diff
@@ -255,6 +255,7 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
         emit ValidatorRemoved(validatorPK, clusterId);
     }
 
+    // WHY NO VALIDATION ON AMOUNT?  WHAT IF NEW POD IS TOO EXPENSIVE AND DOES IMMEDIATE LIQUIDATION OR EVEN SKIPS LIQUIDATION AND GOES TO NEGATIVE
```

Comment:
@andrew-blox  i think transfer is not completed logic, no sense to review it, it's just first impl version